lib/pages/search/public_room/empty_search_public_room_widget_style.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class EmptySearchPublicRoomWidgetStyle {
+  static const double avatarSize = 56.0;
+
+  static const BorderRadiusGeometry avatarBorderRadius =
+      BorderRadius.all(Radius.circular(28.0));
+  static BoxBorder avatarBorder(BuildContext context) {
+    return Border.all(
+      color: Theme.of(context).brightness == Brightness.dark
+          ? const Color(0x222c2d2f)
+          : const Color(0x222c2d2f),
```

Comment:
Here if Theme.of(context).brightness is true or false, the color is the same. Maybe you forgot to change the color in one case ? 
If not you can remove the condition and juste write `const Color(0x222c2d2f)`

Reply:
updated